tests/visualization/test_open3d_vis_utils.py

```diff
@@ -0,0 +1,35 @@
+"""
+Unit tests for Open3d visualizaation code.
+
+Author: John Lambert
+"""
+
+import numpy as np
+import open3d
+
+import visualization.open3d_vis_utils as open3d_vis_utils
+
+
+def test_create_colored_spheres_open3d() -> None:
+    """Try creating spheres centered at randomly generated 3d locations (disclaimer: slow)."""
+    num_points = 1000
+    point_cloud = np.random.randn(num_points, 3)
+    # high bound is exclusive
+    rgb = np.random.randint(low=0, high=256, size=(num_points, 3))
+    sphere_radius = 0.1
+
+    spheres = open3d_vis_utils.create_colored_spheres_open3d(point_cloud, rgb, sphere_radius)
+
+    assert isinstance(spheres, list)
+    assert all([isinstance(sphere, open3d.geometry.TriangleMesh) for sphere in spheres])
+
+
+def test_create_colored_point_cloud_open3d() -> None:
+    """Try creating points at randomly generated 3d locations (should be fast)."""
+    num_points = 10000
+    point_cloud = np.random.randn(num_points, 3)
```

Comment:
maybe a testcase class is better here so you can have the point cloud as a member shared across tests.

Reply:
That's fair. I used a different sized point cloud for each though, since 10k is too many for the sphere rendering (way too slow) but no problem for the point rendering

Reply:
You could just create one point cloud and subsample as needed.

Reply:
That's true. Not sure if it makes a big difference though, i think it's ok.